azure-monitor-opentelemetry/azure/monitor/opentelemetry/util/configurations.py

```diff
@@ -81,6 +86,19 @@ def _default_disable_tracing(configurations):
     configurations[DISABLE_TRACING_ARG] = default
 
 
+def _default_disabled_instrumentations(configurations):
+    disabled_instrumentation = environ.get(
+        OTEL_PYTHON_DISABLED_INSTRUMENTATIONS, []
```

Comment:
Should we add an entry to the README for this configuration?

Reply:
Updated readme